src/SortChecker.cpp

```diff
@@ -44,7 +44,7 @@ class Visitor : public RecursiveASTVisitor<Visitor> {
     return E;
   }
 
-  void replaceCallee(DeclRefExpr *DRE, const char *Replacement) const {
+  void replaceCallee(DeclRefExpr *DRE, llvm::StringRef Replacement) const {
     SourceRange Range = {DRE->getBeginLoc(), DRE->getEndLoc()};
     RW.ReplaceText(Range, Replacement);
```

Comment:
I was originally worried that `Replacement` will be cached inside the rewriter but it looks like it makes a copy so it's ok.